setup.py

```diff
@@ -10,6 +15,23 @@
     'complete': open('complete-requirements.txt').readlines()
 }
 
+class PreSDistNLTKDataUnpackCommand(sdist):
+    """Before creating source distribution, untar nltk data files, so they'll be included via the manifest file.
+
+    Note the usage of `pkg_resources.resource_filename` for pathing assumes you're building the source distribution with your cwd
+    in the repo.
+    """
+    def run(self):
+        nltk_data_tarball_path = pkg_resources.resource_filename('nlp_primitives', str(pathlib.Path('data','nltk-data.tar')))
+        nltk_data_extract_path = pkg_resources.resource_filename('nlp_primitives', str(pathlib.Path('data')))
+        print(f'Extracting nltk data files from {nltk_data_tarball_path} to {nltk_data_extract_path}')
+        print(pkg_resources.resource_listdir('nlp_primitives', str(pathlib.Path('data'))))
+        with tarfile.open(nltk_data_tarball_path, "r") as tar:
+            tar.extractall(path=nltk_data_extract_path)
+            tar.close()
```

Comment:
tar will close automatically when the with  block is done, we can remove this line and the code below can move outside the with block